### PR TITLE
Add docs for asciinema

### LINKED ISSUE #109: provide asciinema/gif demo

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Signed-off-by: lachmanfrantisek <flachman@redhat.com>

Resolves #109 

conu

Any suggestions to the scenario (in the `docs/source/asciinema.rst`)?

Where should be the gif placed? `README.md`?
* `# Docker example` section,
* on the top?

### CODE REVIEW

File: docs/source/asciinema.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+How to make a new gif
+=====================
+
+* Recording: `asciinema rec -w 1 -c=ipython3 conu.json`
```
Comment (TomasTomecek) [inline docs/source/asciinema.rst]: which package we need to install?

Comment (lachmanfrantisek) [reply docs/source/asciinema.rst]: - asciinema (`pip3`/`dnf`)

**Comment by lachmanfrantisek** [comment]
Thank you, I've removed the logging setup if it is not a problem...

**Comment by lachmanfrantisek** [comment]
@dhodovsk good catch, I've made it a little bit longer.

**Comment by dhodovsk** [comment]
Thank you for this! :)

**Pull request merged**